src/pages/questionPage.js

```diff
@@ -54,6 +62,15 @@ export const initQuestionPage = () => {
     }
   }
 
+  //Selecting the progress bar
+  // const progressBarFull = document.getElementById('progressBarFull');
+  // const progressBarIndicator = currentIndex + 1;
+  // progressBarFull.style.width = `${(progressBarIndicator / MAX_QUESTIONS) * 100}%`;
+
+  document
+    .getElementById(NEXT_QUESTION_BUTTON_ID)
+    .addEventListener('click', nextQuestion);
+
```

Comment:
The next question button can be deleted, Emin added two buttons, and one of them makes the same.
```suggestion
```